custom_components/tahoma/config_flow.py

```diff
@@ -86,7 +86,8 @@ async def async_step_user(
                 return await self.async_validate_input(user_input)
             except TooManyRequestsException:
                 errors["base"] = "too_many_requests"
-            except BadCredentialsException:
+            except BadCredentialsException as exception:
+                _LOGGER.debug(type(exception))
```

Comment:
@tetienne I used this temporarily so you can easily check if it uses new somfy auth. Just put in the wrong creds and it will say 'SomfyBadCredentialsException' in log, if new one is used.